### ISSUE TITLE
Support for kibana 5

### ISSUE THREAD

**Issue by sophiec20** [prelert/kibana-swimlane-vis#9]
Swimlane should support Kibana 5

**Comment by pemontto** [comment]
And now it will need to support 5.0.1 which I assume is just a string change somewhere. This version requirement is going to make me very reluctant to ever update Kibana, surely a dot releases shouldn't require this.

```
Plugin installation was unsuccessful due to error "Incorrect Kibana version in plugin [prelert_swimlane_vis]. Expected [5.0.1]; found [5.0.0]"
```

Update: if anyone else comes across this, a bandaid solution is to manually download and unzip to the Kibana plugins directory then edit the package.json and set the version to match that of your Kibana instance.

**Comment by obudiman** [comment]
+1 @pemontto

**Comment by peteharverson** [comment]
I have now published a 5.0.1 release of the plugin.

**Comment by peteharverson** [comment]
Closing issue - releases for Kibana 5.0.0 and 5.0.1 can be found at https://github.com/prelert/kibana-swimlane-vis/releases. Plan will be to publish subsequent 5.0.x releases in step with the elastic stack releases.
